src/components/Layout/Header/NavBar/Native/BackupPassphraseModal/BackupPassphraseInfo.tsx

```diff
@@ -34,7 +34,7 @@ const moduleLogger = logger.child({
 
 export const BackupPassphraseInfo = ({ vault }: { vault: Vault | null }) => {
   const translate = useTranslate()
-  const history = useHistory()
+  const { goBack: handleBackClick, ...history } = useHistory()
```

Comment:
Technically `history` is `Omit<History, 'description'>` but keep things more declarative to still name it history